pkg/kubelet/eviction/helpers.go

```diff
@@ -609,9 +609,9 @@ func podRequest(pod *v1.Pod, resourceName v1.ResourceName) resource.Quantity {
 	for i := range pod.Spec.InitContainers {
 		switch resourceName {
 		case v1.ResourceMemory:
-			containerValue.Add(*pod.Spec.Containers[i].Resources.Requests.Memory())
```

Comment:
i believe you should use `initValue.Add` and not `containerValue.Add`

Reply:
seems that second commit didn't ship, fixed

Reply:
LGTM